'C:\Python27\Python.exe ficheiro-saida.py' returned 0.
